updates/31/asset_collections_updates.sql

```diff
@@ -1,2 +1,678 @@
 INSERT INTO asset_collections(id, name, symbol) VALUES (342, "Morpho", "MORPHO");
 *
+UPDATE asset_collections SET main_asset='eip155:1/erc20:0xB6eD7644C69416d67B522e20bC294A9a9B405B31' WHERE id=0;
+INSERT INTO asset_collections(id, name, symbol) VALUES (0, '0xBitcoin', '0xBTC');
```

Comment:
shouldn't the insert have the main asset also in the query?

Reply:
that'd require the refactor of the assets updater. 
normally, the **UPDATE** statements are sufficient since the data already exists in the database but the **INSERTS** are needed since the tests expect such format.